From e93477202daf1667569f7eb3240a74467f3a871b Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@st.com>
Date: Mon, 30 Nov 2020 17:35:46 +0100
Subject: [PATCH 194/529] ARM: dts: stm32: update sound card on
 stm32mp157xx-dkx

- Rename the sound card for STM32MP15x Disco board to allow SoC
diversity management. Change from STM32MP1-DK to STM32MP15-DK.

- Remove clocks property in SAI subblocks nodes, for STM32MP157xx
Disco board device tree, as this property is already defined
in SoC device tree.
Keep only this property where master clock is added.

Signed-off-by: Olivier Moysan <olivier.moysan@st.com>
Change-Id: I9f2ef288e91343027eaa7688f006d73e3387f9a0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/278992
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
Reviewed-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm/boot/dts/stm32mp15xx-dkx.dtsi | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/arch/arm/boot/dts/stm32mp15xx-dkx.dtsi
+++ b/arch/arm/boot/dts/stm32mp15xx-dkx.dtsi
@@ -78,7 +78,7 @@
 
 	sound {
 		compatible = "audio-graph-card";
-		label = "STM32MP1-DK";
+		label = "STM32MP15-DK";
 		routing =
 			"Playback" , "MCLK",
 			"Capture" , "MCLK",
@@ -507,8 +507,6 @@
 	sai2a: audio-controller@4400b004 {
 		#clock-cells = <0>;
 		dma-names = "tx";
-		clocks = <&rcc SAI2_K>;
-		clock-names = "sai_ck";
 		status = "okay";
 
 		sai2a_port: port {
